packages/common/src/store/buy-usdc/utils.ts

```diff
@@ -1,7 +1,7 @@
 import { call, select } from 'typed-redux-saga'
 
 import { createUserBankIfNeeded } from 'services/audius-backend/solana'
-import { IntKeys } from 'services/index'
```

Comment:
??? how did this ever work?

Reply:
I do remember a time where circular dependencies would cause serious issues. I think our build tooling at this point is advanced enough to work around it and just generate a warning. FWIW, we have a few other long-standing instances of this.
I have worked on teams in the past where we didn't use index files for exactly this reason, preferring direct module imports instead.
Though, in this case, since we're bundling everything up into a "package", the index files serve as a way to join the things we want to be finally exported at the top level.

I would say when we un-package common, we might do well to get rid of index files :-)